flocker/node/agents/blockdevice.py

```diff
@@ -61,10 +61,17 @@
 class DatasetStates(Names):
     """
     States that a ``Dataset`` can be in.
+
     """
+    # Exists, but not attached:
     NON_MANIFEST = NamedConstant()
+    # Attached to this node but no filesystem
+    ATTACHED_NO_FILESYSTEM = NamedConstant()
+    # Attached to this node, has filesystem
     ATTACHED = NamedConstant()
+    # Mounted on this node:
     MOUNTED = NamedConstant()
+    # XXX seems pointless might delete at some point
```

Comment:
Given the docstring of this class it seems that this should exist. Perhaps that docstring should be updated when this is removed. (States that a Dataset with an existing volume can be in or somesuch).